corehq/apps/app_manager/models.py

```diff
@@ -2705,6 +2705,13 @@ def new_form(self, name, lang, attachment=Ellipsis):
         form = self.get_form(-1)
         if attachment == Ellipsis:
             attachment = get_blank_form_xml(name)
+        elif attachment == "default_followup_form":
```

Comment:
The `attachment` parameter is meant to be actual xml content (see a few lines below, where it's assigned to `form.source`). I'd move the context logic and `get_default_followup_form_xml` call into modules.py, passing the generated xml as `attachment` to the `new_form` call there that creates the followup form.